OpenRA.Mods.Common/Traits/Attack/AttackFollow.cs

```diff
@@ -48,7 +48,7 @@ public override Activity GetAttackActivity(Actor self, Target newTarget, bool al
 			return new AttackActivity(self, newTarget, allowMove, forceAttack);
 		}
 
-		protected override void OnStopOrder(Actor self)
+		public override void OnStopOrder(Actor self)
```

Comment:
A specific reason this has no `Some 3rd-party mods rely on this being public` comment, but the other do?

Reply:
Because this just overrides the one from AB. Imagine every Attack* trait would override this, would we want to add the comment each time?